tools/plink_load_variant_file.cwl

```diff
@@ -0,0 +1,69 @@
+class: CommandLineTool
+cwlVersion: v1.2
+id: plink_load_variant_file
+doc: |-
+  Create a plink file from a VCF/BCF
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/danmiller/plink:1.90-b7.2'
```

Comment:
Did we approve these dockers yet? Or will the next update point to bixu?

Reply:
Yes, this docker is now on d3b-bixu. I can update it.